src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -5,7 +5,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>latest</LangVersion>
     <PackageId>Microsoft.Azure.WebJobs</PackageId>
-    <Description>This package contains the runtime assemblies for Microsoft.Azure.WebJobs.Host. It also adds rich diagnostics capabilities which makes it easier to monitor the WebJobs in the dashboard. For more information, please visit http://go.microsoft.com/fwlink/?LinkID=320971</Description>
+    <Description>This package contains the runtime host components of the WebJobs SDK. For more information, please visit https://learn.microsoft.com/en-us/azure/app-service/webjobs-sdk-get-started.</Description>
```

Comment:
We should continue use fwlink for this. If needed, we can create one for the target.

Reply:
Applies to all links.

Reply:
Created a new fwlink https://go.microsoft.com/fwlink/?linkid=2279708 and updated all usages